docs/genai/howto/build-from-source.md

```diff
@@ -38,161 +37,96 @@ These instructions assume you are in the `onnxruntime-genai` folder.
 These instruction use `win-x64`. Replace this if you are using a different architecture.
 
 ```bash
-curl -L https://github.com/microsoft/onnxruntime/releases/download/v1.18.0/onnxruntime-win-x64-1.18.0.zip -o onnxruntime-win-x64-1.18.0.zip
-tar xvf onnxruntime-win-x64-1.18.0.zip
-move onnxruntime-win-x64-1.18.0 ort 
+curl -L https://github.com/microsoft/onnxruntime/releases/download/v1.18.1/onnxruntime-win-x64-1.18.1.zip -o onnxruntime-win-x64-1.18.1.zip
```

Comment:
Should we change the version to `1.19.2` since that's the latest release?

Reply:
Updated

Reply:
The version of ort lib used at build time does not reflect the version of the ort library used at runtime. As such, we only need the ort version that has the API that ort-genai needs.
If possible, we should use old(er) version of the ort lib so that we can support a wider range of ort libs at runtime. I think this will become obsolete when we add auto download and use ort at compile time.